lesson_08/script.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const newElement = document.createElement("div");
+newElement.id = "js-loading";
+ul.before(newElement);
```

Comment:
IMO
The newElement variable here.
I think it would be better to put it in addLoading().

An empty div is generated just for the initial loading, and
When you want to insert another loading image, you can use the
It will not be reusable.

Reply:
I see, I am learning.

Reply:
Thanks for the correction.

IMO
There is a rule that only <li> tags can be placed directly under <ul> tags, so in this case I think it would be better to put it directly under the <body> tag.

The following site is a good reference for html parent-child relationships.
https://yoshikawaweb.com/element/